authorization-service/src/handlers/basicAuthorizer.js

```diff
@@ -0,0 +1,41 @@
+const basicAuthorizer = async (event, ctx, cb) => {
+  console.log('function started, event:', event)
+
+  if (event.type !== 'TOKEN') cb('Unauthorized')
+
+  try {
+    const { authorizationToken } = event
+    const encodedCreds = authorizationToken.split(' ')[1]
```

Comment:
It seems that deployed version expects only the token, without anything before it (like 'Basic ' part). I tried 'Basic %encoded_token%, app returned 403 error

Reply:
Lambda authorizer expects authorizationToken in the format "string token" (word|space|token) token itself is separated by the space. If Authorization header contained only token, without 'Basic' part, it was parsed as the wrong credentials and 403 returned